finality-aleph/src/testing/mocks/validator_network.rs

```diff
@@ -0,0 +1,96 @@
+#![allow(dead_code)]
+use std::sync::Arc;
+
+use aleph_primitives::{AuthorityId, KEY_TYPE};
+use sp_keystore::{testing::KeyStore, CryptoStore};
+
+use crate::{
+    network::{mock::Channel, Data, Multiaddress, NetworkIdentity, PeerId},
+    validator_network::Network,
+};
+
+type MockPeerId = AuthorityId;
```

Comment:
I think you should use unaliased `AuthorityId` everywhere here – it is inherently connected to `validator_network`'s interfaces anyway.